### ISSUE TITLE
Amend during interactive rebase not possible

### ISSUE THREAD

**Issue by Turbo87** [ForkIssues/Tracker#201]
During interactive rebase using the `edit` option it is not possible to amend the previous commit since the commit view only shows the "Continue Rebase" button and the progress bar. Similarly it is not possible to create new commits in that state either.

**Comment by DanPristupov** [comment]
Yes, this is a limitation of current RI implementation. I agree it would be nice to have an ability to amend during rebase.

I have a lot of other active tasks now, so there's no ETA.

**Comment by Turbo87** [comment]
good point, I guess they are indeed related
